src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/LicenceSummary.kt

```diff
@@ -2,13 +2,17 @@ package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
 
 import com.fasterxml.jackson.annotation.JsonFormat
 import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceKind
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
 import java.time.LocalDate
 import java.time.LocalDateTime
 
 @Schema(description = "Response object which summarises a licence")
 data class LicenceSummary(
+  @Schema(description = "kind of licence", example = "CRD")
+  val kind: LicenceKind,
```

Comment:
I've only added subtypes for the detailed licence representation. Licence summaries don't currently expose type specific fields as of yet